lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -393,12 +425,13 @@ spendTx tx !u =
         then collateralInputs tx
         else inputs tx
 
--- | Construct a UTxO corresponding to a given transaction. It is important for
--- the transaction outputs to be ordered correctly, since they become available
--- inputs for the subsequent blocks.
+-- | Construct a 'UTxO' corresponding to a given transaction.
+--
+-- It is important for the transaction outputs to be ordered correctly,
+-- as they become available inputs for subsequent blocks.
```

Comment:
but we have TxIn basically having TxId and index and it is enough to point correctly to any input, right?

Reply:
Yes, it's just that this index depends on the order of outputs in the list of outputs — that's all what this comment intends to say.